cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+class Login_Management {
+
+  static #passwordRecoveryButton = '[data-testid="forgot-password"]'
+  static #usernameLabel = '[data-testid="username-label"]'
+  static #emailInputBox = '[id="username"]'
+  static #infoMessage = '[data-testid="info-message"]'
+  static #submitButton = '[data-testid="btn-submit"]'
+  static #cancelButton = '[data-testid="btn-cancel"]'
+
+  beingOnLoginPage() {
+    cy.visit(Cypress.env('BRB'))
+  }
+
+  clickOnForgotPassword() {
+    cy.get(Login_Management.#passwordRecoveryButton).eq(1).click()
+  }
+
+  showElementOnDialog() {
```

Comment:
Method name is contradicting with the behaviour of the method. Adapt accordingly as per action performing in the method.

Reply:
in this method we are showing the elements on the password dialog, so assertions are related to visibility of label, input box, button ..etc in this method, may be put an example here so I'll update the method name starting with 'verb' ?

Reply:
updated the method name to  'showUpElementsOnDialog'.